internal/driver/commands.go

```diff
@@ -254,7 +270,7 @@ func usage(commandLine bool) string {
 		prefix = "-"
 	}
 	fmtHelp := func(c, d string) string {
-		return fmt.Sprintf("    %-16s %s", c, strings.SplitN(d, "\n", 2)[0])
+		return fmt.Sprintf("    %-22s %s", c, strings.SplitN(d, "\n", 2)[0])
```

Comment:
Curious: why did 16 change to 22?

Reply:
To accommodate extra texts (e.g. =<f>) added to the help message and really long option name (i.e. `relative_percentages`)